dashboard/config/application.rb

```diff
@@ -37,6 +37,12 @@ class Application < Rails::Application
 
     config.middleware.insert_after Rails::Rack::Logger, VarnishEnvironment
     config.middleware.insert_after VarnishEnvironment, FilesApi
+
+    if CDO.throttle_data_apis
```

Comment:
Actually, Redis *is* available in non-prod environments, since it is already required by Netsim.  (Even our developer instructions ask you to install the service, although not all developers have done this.)